### ISSUE TITLE
Azure PaaS compatibility?

### ISSUE THREAD

**Issue by saberone** [Sitecore/SharePoint-Connect#2]
Is this compatible with Azure PaaS?

**Comment by michaelthyregod** [comment]
I don't think it is. It is the same code as the last official release from Sitecore, which I can't remember if supports Azure Paas. But you are welcome to do pull requests, or clone it. Be aware that is uses old 2013 Sharepoint components, but it could easily be upgraded to use Microsoft.SharePoint2013.CSOM or Microsoft.SharePointOnline.CSOM instead I think. Hope it helps a bit.
